packages/components/src/RecurringSelect/components/DayOfMonthSelect/DayOfMonthSelect.css

```diff
@@ -0,0 +1,84 @@
+/* 
+  faking a border with a background on the parent element, gaps & padding
+  to prevent double borders on overlapping boxes
+*/
+.container {
+  display: grid;
+  width: 100%;
+  padding: var(--space-smaller);
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-large);
+  grid-template-areas:
+    ". . . . . . ."
+    ". . . . . . ."
+    ". . . . . . ."
+    ". . . . . . ."
+    ". . . lastday lastday lastday lastday";
+  row-gap: var(--space-smaller);
+  column-gap: var(--space-smaller);
+}
+
+.checkboxWrapper {
+  position: relative;
+}
+
+.checkboxWrapper:nth-child(n + 1):nth-last-child(-n + 1) {
+  grid-area: lastday;
+}
+
+/* kept present but hidden for accessibility */
+.container input[type="checkbox"] {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 1rem;
+  height: 1rem;
+  opacity: 0;
+}
+
+.dayCheckbox {
+  display: block;
+  padding: 1rem;
+  border-radius: var(--radius-large);
+  text-align: center;
+  user-select: none;
+  background: var(--color-white);
+  transition: 300ms;
```

Comment:
Can we use `--timing-base` here to be consistent with how we handle similar visual transitions in Button, etc?

Reply:
```suggestion
  transition: var(--timing-base);
```